tests/rapid_gossip_sync_test.rs

```diff
@@ -26,84 +27,112 @@ mod zero_conf_test {
     fn test_update_from_0_and_partial_update() {
         let node_handle = NodeHandle::new_with_lsp_rgs_setup();
 
-        let node = node_handle.start().unwrap();
-        let lipa_node_id = node.get_node_info().node_pubkey.to_hex();
-        assert_eq!(node.get_node_info().num_peers, 1);
-
         let lspd_node_id = nigiri::query_node_info(NodeInstance::LspdLnd)
             .unwrap()
             .pub_key;
 
-        // CONNECT NigiriLnd -> LspdLnd -> 3L + NigiriCln -> LspdLnd
-        nigiri::node_connect(
-            NodeInstance::NigiriLnd,
-            &lspd_node_id,
-            LSPD_LND_HOST,
-            LSPD_LND_PORT,
-        )
-        .unwrap();
-        nigiri::node_connect(
-            NodeInstance::NigiriCln,
-            &lspd_node_id,
-            LSPD_LND_HOST,
-            LSPD_LND_PORT,
-        )
-        .unwrap();
-        sleep(Duration::from_secs(20));
-
-        nigiri::lnd_node_open_channel(NodeInstance::LspdLnd, &lipa_node_id, false).unwrap();
-        nigiri::cln_node_open_pub_channel(NodeInstance::NigiriCln, &lspd_node_id).unwrap();
-        try_cmd_repeatedly!(nigiri::mine_blocks, N_RETRIES, HALF_SEC, 10);
-        wait_for_new_channel_to_confirm(NodeInstance::LspdLnd, &lipa_node_id);
-        wait_for_new_channel_to_confirm(NodeInstance::NigiriCln, &lspd_node_id);
-
-        assert_eq!(node.get_node_info().channels_info.num_channels, 1);
-        assert_eq!(node.get_node_info().channels_info.num_usable_channels, 1);
-
-        assert!(node.get_node_info().channels_info.inbound_capacity_msat > 2 * HUNDRED_K_SATS);
-
-        // Pay from NigiriCln to 3L to create outbound liquidity
-        let invoice_cln = node
-            .create_invoice(HUNDRED_K_SATS, "test".to_string())
+        {
+            let node = node_handle.start().unwrap();
+            let lipa_node_id = node.get_node_info().node_pubkey.to_hex();
+            assert_eq!(node.get_node_info().num_peers, 1);
+
+            // CONNECT NigiriLnd -> LspdLnd -> 3L + NigiriCln -> LspdLnd
```

Comment:
```suggestion
            // Setup channels:
            // NigiriLND -> LspdLnd  -> 3L
            // NigiriCLN  🡥 
```

Would that be accurate (and more explicative)?

Reply:
Yep, definitely better. Applied in [363abde](https://github.com/getlipa/lipa-lightning-lib/pull/164/commits/363abde066602aae0ce84327883736494af54735)